pkg/discovery/detectors/detector_config_test.go

```diff
@@ -230,7 +232,7 @@ var masterNodeTests = []MasterNodeTestCase{
 func TestDetectorConfig_detectMasterNode(t *testing.T) {
 	for scenarioIndex, scenarioName := range []string{"validFullConfig", "missingMasterNodeDetectors",
 		"missingDaemonPodDetectors", "missingBoth", "missingList", "emptySection"} {
-		setupTest(t, scenarioName)
+		_, _ = setupTest(scenarioName)
```

Comment:
Is the `_, _` necessary here?

Reply:
This is not necessary but nice to have IMO for the following reasons:
* Get rid of the unhandled error code inspection warning in IntelliJ IDE
* Communicate to the code reader that we are deliberately ignoring a potential error